common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/AccountManagerAddAccountStrategy.java

```diff
@@ -62,9 +62,12 @@ public AccountManagerAddAccountStrategy(final Context context) {
     }
 
     @Override
-    @Nullable public Bundle communicateToBroker(final @NonNull BrokerOperationBundle brokerOperationBundle)
+    @Nullable
+    public Bundle communicateToBroker(final @NonNull BrokerOperationBundle brokerOperationBundle)
             throws BrokerCommunicationException {
-        final String methodName = brokerOperationBundle.getOperation().name();
+        final String methodName = ":communicateToBroker";
+        final String operationName = brokerOperationBundle.getOperation().name();
+        Logger.info(TAG + methodName, "Broker operation: " + operationName);
```

Comment:
Also include the "broker package name" here, so that we know which is the active broker.
Do that for all 3 ipc strategies

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8